Docker/start-container.sh

```diff
@@ -3,14 +3,14 @@
 #Set user and group
 umask 0002
 PUID=${PUID:-`id -u squeezeboxserver`}
-PGID=${PGID:-`id -g squeezeboxserver`}
+PGID=${PGID:-`id -g users`}
 
 usermod -o -u "$PUID" squeezeboxserver
-groupmod -o -g "$PGID" squeezeboxserver
+groupmod -o -g "$PGID" users
 
 umask 0002
 
 #Add permissions
-chown -R squeezeboxserver:users /config /playlist
+chown -R squeezeboxserver:users /config /playlist /lms
 
-/usr/bin/perl /lms/slimserver.pl --user squeezeboxserver --prefsdir /config/prefs --logdir /config/logs --cachedir /config/cache
+su squeezeboxserver -c /usr/bin/perl /lms/slimserver.pl --prefsdir /config/prefs --logdir /config/logs --cachedir /config/cache
```

Comment:
Here we need to change how arguments are passed to perl.
It should bi like this: perl script.pl arg1 arg2

Reply:
Put quotes around the perl command line.